CRM/Contact/BAO/Query.php

```diff
@@ -3288,7 +3288,7 @@ public function tag(&$values) {
 
     $etTable = "`civicrm_entity_tag-" . uniqid() . "`";
 
-    if ($useAllTagTypes[2]) {
+    if (!empty($useAllTagTypes[2])) {
```

Comment:
ok - seems odd we never see enotices on this already though?

Reply:
Yeh this one is the odd one i dunno why